src/_locales/en/messages.json

```diff
@@ -346,5 +346,26 @@
     "welcome_1b_dragging": "Drag your key backup into this dialog…",
     "welcome_1b_file_selected": "Selected exported key backup: {{filename}}",
     "untitled_wallet": "Untitled Wallet",
-    "your_message": "Your message…"
+    "your_message": "Your message…",
+    "plugin_file_service_display_name": "Plugin: File Service",
+    "plugin_file_service_file_key_copied": "File key copied.",
+    "plugin_file_service_on_insert": "Use the file",
+    "plugin_file_service_uploading_preparing": "Preparing",
+    "plugin_file_service_uploading_estimating_time": "Estimating time…",
+    "plugin_file_service_uploading_in_minute_remaining": "{{seconds}}s remaining",
+    "plugin_file_service_uploading_remaining": "{{minutes}}m {{seconds}}s remaining",
+    "plugin_file_service_uploading_on_cancal": "Uploading. Please do not close this page.",
+    "plugin_file_service_show_more": "Show More",
+    "plugin_file_service_on_encrypt_it": "Make It Encrypted",
+    "plugin_file_service_on_show_more": "NOT SUPPORTED",
+    "plugin_file_service_legal_text": "By using this plugin, you agree to the <terms>terms</terms> and the <policy>privacy policy</policy>.",
+    "plugin_file_service_legal_terms_link": "https://legal.maskbook.com/arweave/file-service/plugin-terms.html",
+    "plugin_file_service_legal_policy_link": "https://legal.maskbook.com/arweave/file-service/privacy-policy-uploader.html",
+    "plugin_file_service_drop_indicator": "Drop to upload",
+    "plugin_file_service_drop_hint_1": "Size limit: {{limit}}.",
+    "plugin_file_service_drop_hint_2": "The file will be uploaded immediately and cannot be cancelled.",
+    "plugin_file_service_drop_here": "Drop a file here to upload",
+    "plugin_file_service_error_101": "The input is not a single file.",
+    "plugin_file_service_error_102": "The file is too large; limit is {{limit}}.",
```

Comment:
```suggestion
    "plugin_file_service_error_102": "The file is too large. The max size of {{limit}}.",
```

Reply:
`The max size is {{limit}}`

Reply:
@neruthes

Reply:
I suggest that every error message should consist of 1 sentence only, unless otherwise necessary. For this specific case, using semicolon should be favorable, while the exact words can be changed.

In my view, "...; the limit is" and "...; the max size is" are both fine. I would like to leave both options for you. Feel free to commit any between them.